### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by ceholden** [ceholden/yatsm#3]
### Algorithm
- [x] Document parameters for `yatsm.py`
### Dataset preparation
- [ ] Stacked images for now... and probably for good.
### Exploration
- [ ] `run_yatsm.py`
- [ ] TSTools interface
### Batch process
- [ ] How to setup the input data for `line_yatsm.py`
- [x] How to create config .ini file for `line_yatsm.py`
- [x] How to run `line_yatsm.py` in parallel for SGE cluster environment
### Output visualization
- [x] How to create changemaps and such from `scripts/yatsm_changemap.py`
  - Wrote documentation with basic text explanation of `yatsm_changemap.py`
  - More TODO:
    - [ ] Example images
    - [ ] Explanation of --root, --result, and `--image` argument flags
- [x] Creation of prediction maps, coefficient images, land cover maps
  - Wrote documentation with basic text explanation of `yatsm_map.py`
  - More TODO:
    - [ ] Example maps
    - [ ] Images helping explain `--after` and `--before`
    - [ ] More information on CLI flags / switches
    - [ ] Explanation of `--root`, `--result`, and `--image` argument flags
### Classification
- [ ] Training
- [ ] Batch prediction
### Phenology
- [x] Link to Eli's paper
- [ ] Description of algorithm outputs
- [ ] Description of departures from his algorithm
- [ ] Description of algorithm parameters
- [ ] Future work -- year to year departures from long term mean phenology
## Hosting

Currently hosted on `readthedocs`
